Sources/ArgumentParserTestHelpers/TestHelpers.swift

```diff
@@ -136,7 +136,7 @@ extension XCTest {
   public func AssertExecuteCommand(
     command: String,
     expected: String? = nil,
-    exitCode: Int32 = 0,
+    exitCode: Int32 = ExitCode.success.rawValue,
```

Comment:
I think you should use directly `ExitCode` as a type for `AssertExecuteCommand` - to force you to keep tests with the same semantic and if it is the output for the execution, it will be always the same type. 

If you use `Int32` you will be less strict, and some bugs can be introduced. wdut?

```suggestion
    exitCode: ExitCode = .success,
```

Reply:
👍 I was resisting this for some reason. Thanks!